qemu/tests/live_snapshot_transaction.py

```diff
@@ -0,0 +1,43 @@
+import re
+import logging
+
+from autotest.client.shared import error
+from qemu.tests import live_snapshot_basic
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    live_snapshot_transaction test:
+
+    1. Boot up guest with a system disk and 2 data disk.
+    2. Create multiple live snapshots simultaneously for all 3 disks with transaction.
+    3. Check guest which should boot up and reboot successfully.
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    image_chain = params.get("images")
+    image_chain = re.split("\s+", image_chain)
+    arg_list = []
+    try:
+        for index, image in enumerate(image_chain):
+            image_params = params.object_params(image)
+            transaction_test = live_snapshot_basic.LiveSnapshot(test,
+                                                                image_params,
+                                                                env, image)
+            transaction_test.snapshot_args.update({"device": transaction_test.device})
+            transaction_test.snapshot_file = image + "-snap"
+            snapshot_file = transaction_test.get_snapshot_file()
+            transaction_test.snapshot_args.update({"snapshot-file": snapshot_file})
+            args = {"type": "blockdev-snapshot-sync",
+                    "data": transaction_test.snapshot_args}
+            arg_list.append(args)
+
+        error_context.context("Create multiple live snapshots simultaneously"
+                              " with transaction", logging.info)
+        transaction_test.vm.monitor.transaction(arg_list)
```

Comment:
Hi qizhu,

Updated, could you hel preview it again?

Thanks.

15:22:08 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.live_snapshot.transaction